tutorials/rise_imagenet.ipynb

```diff
@@ -7,7 +7,7 @@
    "source": [
```

Comment:
<p>Reads well :-)</p><p>Formatting: there are 3 double spaces in the second paragraph of this block.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/210/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>